Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/outbound/deferred/request/StandardOutboundPatientDiscoveryDeferredRequestTest.java

```diff
@@ -285,15 +290,76 @@ public void policyFailed() throws ConnectionManagerException {
         verify(policyChecker).checkOutgoingPolicy(requestArgument.capture());
         assertEquals(firstTargetRequest, requestArgument.getValue().getPRPAIN201305UV02());
 
-        verify(auditLogger).auditRequestMessage(eq(request), eq(assertion), any(NhinTargetSystemType.class),
+        verify(mockEJBLogger).auditRequestMessage(eq(request), eq(assertion), any(NhinTargetSystemType.class),
             eq(NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION), eq(NhincConstants.AUDIT_LOG_NHIN_INTERFACE),
-            eq(Boolean.TRUE), isNull(Properties.class), eq(NhincConstants.PATIENT_DISCOVERY_DEFERRED_REQ_SERVICE_NAME));
+            eq(Boolean.TRUE), isNull(Properties.class), eq(NhincConstants.PATIENT_DISCOVERY_DEFERRED_REQ_SERVICE_NAME),
+            any(PatientDiscoveryDeferredRequestAuditTransforms.class));
     }
 
-    private NhinTargetSystemType createNhinTargetSystemType(String hcid) {
-        NhinTargetSystemType target = new NhinTargetSystemType();
-        target.setHomeCommunity(getHomeCommunity(hcid));
-        return target;
+    @Test
+    public void auditOffForOutboundPDDeferredReq() throws ConnectionManagerException {
+        List<UrlInfo> urlInfoList = createUrlInfoList("1.1", "2.2");
+
+        // Mocks
+        PatientDiscovery201305Processor pd201305Processor = mock(PatientDiscovery201305Processor.class);
```

Comment:
Can we pull all these declarations at class level instead of having declared in both methods?